### PR TITLE
LPS-155466, LPS-155457 | master

### PR DESCRIPTION
**Pull request by chaowang-123**
**Background**
Create automated tests for the Frontend Dataset component.

**Test Plan**
**FDSQuickActions#CanExecuteAnAction**
Given: frontend-data-set-sample-web deployed
And Given: FDS Sample Portlet added to page
When: Hover over 1st row item (Sample 1)
And When: click on Pencil Icon 
Then: #test-pencil is appended to browser URL

**FDSQuickActions#CanDisappearWhenNotHovered**
Given: frontend-data-set-sample-web deployed
And Given: FDS Sample Portlet added to page
When: Hover over 1st row item (Sample 1)
And When: hover mouse off of the table body (can hover mouse on table header)
Then: quick action menu is not visible

**JIRA Ticket**:
https://issues.liferay.com/browse/LPS-155466
https://issues.liferay.com/browse/LPS-155457

### CODE REVIEW

**Comment by chaowang-123** [comment]
ci:test:frontend:nocompile

**Comment by chaowang-123** [comment]
ci:test:sf

**Comment by chaowang-123** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
ci:stop

**Comment by susanchen3** [comment]
ci:test:frontend

**Comment by john-co** [comment]
ci:test:frontend:nocompile

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSFilters.testcase [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONLayout.deletePublicLayout(
+				groupName = "Guest",
+				layoutName = "Frontend Data Set Test Page");
+		}
+	}
+
+}
```
Comment (john-co) [inline modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSFilters.testcase:48]: This looks outside the scope of LPS-155457 which is about FDSQuickActions. Maybe meant to add this to a different branch/PR?

Comment (chaowang-123) [reply modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSFilters.testcase]: Hi, this commit had been reverted, thanks!

**Comment by chaowang-123** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
HTTP 500 Internal Server Error error, gonna try again... 

Although FDS Sample Portlet is having loading issues on CI. Related to [LPS-155130](https://issues.liferay.com/browse/LPS-155130), most likely should work that out first before proceeding

**Comment by susanchen3** [comment]
ci:test:frontend:nocompile

**Comment by susanchen3** [comment]
`LIFERAY_ERROR: Problem {detail=null, status=Internal Server Error, title=HTTP 500 Internal Server Error, type=null}` again... will have to work out FDS sample portlet issues first..

**Comment by chaowang-123** [comment]
ci:test:frontend:nocompile

**Comment by chaowang-123** [comment]
ci:test:frontend:nocompile
